Analysis/include/SampleDescriptor.h

```diff
@@ -39,14 +41,39 @@ struct AnalyzerSetup {
     std::string plot_cfg, plot_page_opt, unc_cfg;
     JetOrdering jet_ordering;
     std::map<Channel, std::vector<std::string>> trigger;
-
     std::map<SelectionCut,analysis::EllipseParameters> massWindowParams;
+    std::map<std::string, std::vector<std::string>> limit_setup_raw;
+    std::map<std::string, std::map<EventCategory, std::string>> limit_setup;
 
     bool IsSignal(const std::string& sample_name) const
     {
         const auto iter = std::find(signals.begin(), signals.end(), sample_name);
         return iter != signals.end();
     }
+
+    void CreateLimitSetups()
+    {
+        for(const auto& m: limit_setup) {
+            auto variable_categories_map = m.second;
```

Comment:
Why you initializing map with vector? Is it compiles? Anyhow, it is wrong. Remove this line.